### PR TITLE
added ansible-test jobs for ovs

### PR DESCRIPTION
**Pull request by komaldesai13**
added ansible-test and removed ansible-ee for OVS

### CODE REVIEW

File: zuul.d/project-templates.yaml [hunk 1]
```diff
@@ -1743,11 +1743,11 @@
...
-        - ansible-ee-integration-ovs-stable2.12
-        - ansible-ee-integration-ovs-stable2.11
-        - ansible-ee-integration-ovs-stable2.10
-        - ansible-ee-integration-ovs-stable2.9
+        - ansible-test-integration-ovs-latest
+        - ansible-test-integration-ovs-stable2.12
+        - ansible-test-integration-ovs-stable2.11
+        - ansible-test-integration-ovs-stable2.10
+        - ansible-test-integration-ovs-stable2.9
         - build-ansible-collection:
```
Comment (trishnaguha) [inline zuul.d/project-templates.yaml:1751]: stable-2.13 test job is required

File: zuul.d/openvswitch-openvswitch-jobs.yaml [hunk 2]
```diff
@@ -52,45 +52,49 @@
...
-    name: ansible-ee-integration-ovs-latest
-    parent: ansible-ee-tests
+    name: ansible-test-integration-ovs-latest
+    parent: ansible-test-integration-base
     pre-run:
-      - playbooks/ansible-ee/integration/pre.yaml
-      - playbooks/ansible-ee/openvswitch/pre.yaml
-    run: playbooks/ansible-ee/integration/run.yaml
+      - playbooks/ansible-network-openvswitch-appliance/pre.yaml
+    run: playbooks/ansible-network-openvswitch-appliance/run.yaml
```
Comment (goneri) [inline zuul.d/openvswitch-openvswitch-jobs.yaml:61]: This run.yaml is actually empty.
You should instead run `playbooks/ansible-test-base/run.yaml`

Comment (goneri) [inline zuul.d/openvswitch-openvswitch-jobs.yaml:61]: You can also add: `post-run: playbooks/ansible-test-base/post.yaml`

Comment (goneri) [reply zuul.d/openvswitch-openvswitch-jobs.yaml:61]: You can use ansible-test-cloud-integration-vcenter as an example.

**Review by goneri — CHANGES_REQUESTED** [review]

**Comment by komaldesai13** [comment]
recheck

**Pull request closed without merging**